Documentation/ceph-csi-drivers.md

```diff
@@ -117,49 +151,24 @@ kubectl exec -ti -n rook-ceph rook-ceph-mon-a-6c4f9f6b6-rzp6r -- bash -c "ceph -
 
 Then create a Secret using admin and `kubernetes` keyrings:
 
-## rbd-secret.yaml
+In [secret](../cluster/examples/kubernetes/ceph/csi/example/rbd/secret.yaml), you
+need your Ceph admin/user password encoded in base64. Run `ceph auth ls` in one
+of your Ceph pods, to encode the key of your admin/user run `echo KEY |base64`
```

Comment:
This is a run-on sentence. Mind breaking it up?

Also, would you please provide a complete example command (kubectl exec?) so users know exactly where and how run `ceph auth ls`?

Reply:
@travisn @leseb am trying to create a  new user as  @rootfs suggested [here](https://github.com/rook/rook/pull/2934/files#diff-8be52e0c33719742b3b5ee58f5ae6f74L115)
is this command still valid or am i missing something?
```
$ ./kubectl -nrook-ceph exec -it rook-ceph-mon-a-67f5cb85c8-dgpjg -- bash -c "ceph -c /var/lib/rook/rook-ceph/rook-ceph.config auth get-or-create-key client.kubernetes mon \"allow profile rbd\" osd \"profile rbd pool=rbd\""
2019-04-09 07:23:43.039 7f974ce26700 -1 Errors while parsing config file!
2019-04-09 07:23:43.039 7f974ce26700 -1 parse_file: cannot open /var/lib/rook/rook-ceph/rook-ceph.config: (2) No such file or directory
Error initializing cluster client: ObjectNotFound('error calling conf_read_file',)
command terminated with exit code 1

###file or host  where  am running kubectl
$ cat /var/lib/rook/rook-ceph/rook-ceph.config
[global]
fsid                      = 0905f0ba-b9b2-4550-adcb-244f05b9a0d1
run dir                   = /var/lib/rook/rook-ceph
mon initial members       = a b c
mon host                  = 10.96.96.18:6789,10.106.223.0:6789,10.103.185.118:6789
log file                  = /dev/stderr
mon cluster log file      = /dev/stderr
public addr               = 172.17.0.10
cluster addr              = 172.17.0.10
mon keyvaluedb            = rocksdb
mon_allow_pool_delete     = true
mon_max_pg_per_osd        = 1000
debug default             = 0
debug rados               = 0
debug mon                 = 0
debug osd                 = 0
debug bluestore           = 0
debug filestore           = 0
debug journal             = 0
debug leveldb             = 0
filestore_omap_backend    = rocksdb
osd pg bits               = 11
osd pgp bits              = 11
osd pool default size     = 1
osd pool default min size = 1
osd pool default pg num   = 100
osd pool default pgp num  = 100
crush location            = root=default host=minikube
rbd_default_features      = 3
fatal signal handlers     = false

[client.admin]
keyring = /var/lib/rook/rook-ceph/client.admin.keyring
```

am not sure how this works, do `/var/lib/rook/rook-ceph/rook-ceph.config` shared from the host with mon pods?

Reply:
You have to run this command from either the operator or a toolbox. A mon pod does not have any ceph.conf that has the mon address, everything is passed on the CLI.